### DISCUSSION TITLE
Lists with `ui.isHidden` resolving to `true` - route can still be accessed in the admin UI

### DISCUSSION THREAD

**Discussion by glennflanagan** [keystonejs/keystone#8908] [Feature requests]
## Reproduction steps
1. Create List with `ui.isHidden` function set to resolve to `true`. e.g.:
```
const User = list({
  ui: {
    isHidden: () => true,
  },
  fields: ...
```
2. See the list is hidden from the dasboard and navigation
3. Enter the route to the hidden list into the browser address bar e.g. `http://localhost:3000/users` and observe that you are directed to the list page.

<img width="1904" alt="Screenshot 2022-09-22 at 14 54 45" src="https://user-images.githubusercontent.com/5485364/191768192-87d79660-91c1-4474-92e6-bb2b71175e27.png">

## Expected behaviour
Accessing the route of a list with `ui.isHidden` resolving to `true` should redirect the user back to the Dashboard

**Comment by IgorKha** [comment]
I think this should be solved through the implementation of access rights

**Comment by dcousens** [comment]
The problem now is many developers rely on this behaviour to retain functionality, while removing it from the primary navigation.
We need to support omitting the routes using a different configuration.

Moving this to a feature request for now

**Reply by kennedybaird** [reply]
Can you elaborate on which functionality you are meaning?
